### ISSUE TITLE
Web: Drawer navigation have bad performance in web

### ISSUE THREAD

**Issue by hosseinmd** [react-navigation/react-navigation#9033]
**Current Behavior**
Web: Drawer navigation have bad performance in web

**Expected Behavior**
Expected to have good performance,

**How to reproduce**

I think we could use CSS transform for drawer

**Your Environment**

| software                       | version |
| ------------------------------ | ------- |
| iOS or Android                 |
| @react-navigation/native       |
| @react-navigation/drawer       |
| react-native-reanimated        |
| react-native-gesture-handler   |
| react-native-safe-area-context |
| react-native-screens           |
| react-native                   |
| expo                           |
| node                           |
| npm or yarn                    |

**Comment by hosseinmd** [comment]
I solved that with a copy of the drawer, I removed animated a used style transition to animation. now performance is so better.

**Comment by MohammadAzimi** [comment]
@hosseinmd Could you please explain it more.
I have a problem with drawer swipes, when closing the drawer.

**Comment by hosseinmd** [comment]
This is just about performance of drawer on the web.
@MohammadAzimi  Is this your problem?

**Comment by MohammadAzimi** [comment]
@hosseinmd
Yes, it is about performance. Closing the drawer by swiping doesn't work correctly. I want to disable swiping or make it better.
When I try to close the drawer by swiping, it stands open half-open.
